config/200-serviceaccount.yaml

```diff
@@ -30,3 +30,13 @@ metadata:
     app.kubernetes.io/component: webhook
     app.kubernetes.io/instance: default
     app.kubernetes.io/part-of: tekton-pipelines
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: tekton-events-controller
+  namespace: tekton-pipelines
+  labels:
+    app.kubernetes.io/component: controller
```

Comment:
```suggestion
    app.kubernetes.io/component: events-controller
```

Reply:
Or it looks like you're using `events` elsewhere. No preference, but we should be consistent and not use the existing `controller` component.